src/components/ItemBook.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/booksSlice';
+
+function MyBook({ book }) {
+  const dispatch = useDispatch();
+  // const { title, author } = props;
+  return (
+    <div className="mybook">
+      <div className="title">
+        Title:
+        {book.title}
+      </div>
+      <div className="author">
+        Author:
+        {book.author}
+      </div>
+      <div className="book-option">
+        <button type="button" className="comments-btn">
+          Comments
+        </button>
+        <div className="divider" />
+        <button
+          type="button"
+          className="remove-btn"
+          onClick={() => dispatch(removeBook(book.item_id))}
+        >
+          Removed
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you correct the spelling of `Removed` here to `Remove`